### PR TITLE
bpo-46128: Strip IsolatedAsyncioTestCase frames from reported stacktraces

### PR DESCRIPTION
**Pull request by asvetlov**
https://bugs.python.org/issue46128

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2021-12-19-10-47-24.bpo-46128.Qv3EK1.rst
Comment (kumaraditya303) [inline Misc/NEWS.d/next/Library/2021-12-19-10-47-24.bpo-46128.Qv3EK1.rst]:
```suggestion
Strip :class:`unittest.IsolatedAsyncioTestCase` stack frames from reported
```

**Review by kumaraditya303 — APPROVED** [review]

File: Misc/NEWS.d/next/Library/2021-12-19-10-47-24.bpo-46128.Qv3EK1.rst
Comment (asvetlov) [reply Misc/NEWS.d/next/Library/2021-12-19-10-47-24.bpo-46128.Qv3EK1.rst]: Thanks!

**Comment by asvetlov** [comment]
Unfortunately, it doesn't hide internal `run_until_complete()` call but hides actual test lines :(

**Review by serhiy-storchaka — APPROVED** [review]

**Comment by ambv** [comment]
This missed the boat for inclusion in Python 3.9 which accepts security fixes only as of today.

**Comment by smontanaro** [comment]
(I'm working my way through some PRs which have been approved and are labeled "awaiting merge", hence my seemingly bolt from the blue comment. Why? [Read here](https://discuss.python.org/t/prs-awaiting-merge/21313).)

It appears that @ezio-melotti removed the ``CLA signed`` label back in July. Should the ``awaiting merge`` label also be removed as a side-effect, or does removing ``CLA signed`` not also imply ``CLA not signed``?

**Comment by ezio-melotti** [comment]
The `CLA signed' and `CLA not signed` labels have been removed and replaced by the CLA bot.  I updated the branch to trigger the bot, and it now passed the check.

**Comment by hugovk** [comment]
This missed the boat for inclusion in Python 3.10 which accepts security fixes only as of this week.

This PR is marked as a draft, but has two approvals from core devs. Is it ready for merge?

**Comment by asvetlov** [comment]
Guys, the PR was useless until Python 3.11, which have rewritten `IsolatedAsyncioTestCase`. 
It is based on asyncio.Runner now.
So, adding `__unittest = True` line works well now.

I suggest backporting to 3.12/3.13 because the backport cannot break anything, but I'm fine with merging to the master only if @Yhg1s has objections.

**Review by kumaraditya303 — APPROVED** [review]

**Pull request merged**
